.github/workflows/linting_only.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         black_args: "--check -l 100"
         # stop the build if there are Python syntax errors or undefined names
         flake8: true
-        flake8_args: "scout/ --count --select=E9,F63,F7,F82 --show-source --statistics"
+        flake8_args: "scout/ --count --select=E9,F63,F7,F82 --max-line-length 100 --show-source --statistics"
```

Comment:
Why this change now? The action was failing?

Reply:
Only to get all the info in from the `setup.cfg` file, which is no longer active!

Reply:
(And because the flake8 maintainer is strongly opposed to using toml project files for config, so you can config it in any number of ways except with the project toml. 🙄 You can read on their page if you like.. We should perhaps just switch to `ruff` instead. It's really nice, well behaved and super fast.